ui/components/multichain/address-copy-button/index.scss

```diff
@@ -0,0 +1,11 @@
+.multichain-address-copy-button {
+  border-radius: 999px;
```

Comment:
You should be able to use `borderRadius={BorderRadius.pill}` here instead of this css. It === `9999px`

Reply:
Updated in: https://github.com/MetaMask/metamask-extension/pull/18153/commits/d2a452a708541f78cdb28e3f294a66a8e493604c